### PR TITLE
Meeting Minutes April 11th

### PR DESCRIPTION
**Pull request by MorgaineBarter**
Meeting Minutes April 11th

### CODE REVIEW

**Comment by DocHolidayHunter** [comment]
Looks good

**Review by DocHolidayHunter — COMMENTED** [review]
Looks good

File: docs/Splashkit/Applications/Meeting Minutes/MM11April.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+Monday: 11/04/2022
+
+Attendees at 4:40: [Lily-yyyy], Nick Agiazis, Anthony George, Sarah Gosling, Riley Nicholas Dellios, [MorgaineBarter], Bella Chhour (at 5:10)
+
+Absent: Huy Pham
+
+Next meeting: 14/4/2022 6pm Thursday
+
+Minutes by : [MorgaineBarter] and [AnthonyGeorge]
```
Comment (talzid) [inline docs/Splashkit/Applications/Meeting Minutes/MM11April.md:11]: Curious as to what the brackets mean?

**Review by talzid — APPROVED** [review]

File: docs/Splashkit/Applications/Meeting Minutes/MM11April.md (hunk 1 above)
Comment (DocHolidayHunter) [reply docs/Splashkit/Applications/Meeting Minutes/MM11April.md:11]: I think that it was from the template given. They had brackets for next to the names

**Review by DocHolidayHunter — APPROVED** [review]

**Pull request merged**